tests/integration/components/form-controls/component-test.ts

```diff
@@ -0,0 +1,83 @@
+import { render } from '@ember/test-helpers';
+import a11yAudit from 'ember-a11y-testing/test-support/audit';
+import Changeset from 'ember-changeset';
+import lookupValidator from 'ember-changeset-validations';
+import setupMirage from 'ember-cli-mirage/test-support/setup-mirage';
+import { setupRenderingTest } from 'ember-qunit';
+import hbs from 'htmlbars-inline-precompile';
+import { module, test } from 'qunit';
+
+import { click } from 'ember-osf-web/tests/helpers';
+
+import { nodeValidation } from './validation';
+
+module('Integration | Component | form-controls', hooks => {
+    setupRenderingTest(hooks);
+    setupMirage(hooks);
+
+    test('it renders 991', async function(assert) {
+        const model = {
+            title: '',
+            description: '',
+        };
+        const changeset = new Changeset(model, lookupValidator(nodeValidation), nodeValidation);
+        this.set('changeset', changeset);
+
+        await render(hbs`
+            <form data-test-form>
+                <FormControls as |form| >
+                    <form.text @label='title' @valuePath='title' />
+                    <form.text @label='description' @valuePath='description' />
+                </FormControls>
+            </form>
+        `);
+        assert.dom('[data-test-form]').exists();
+        assert.dom('.form-group > input').exists({ count: 2 });
```

Comment:
`.form-group` is bootstrap-specific, instead we should add data-test selectors to the `form.text` invocations and check for existence of the elements within these, e.g.:
```suggestion
        assert.dom('[data-test-title-input] input').exists({ count: 1 });
        assert.dom('[data-test-description-input] input').exists({ count: 1 });
```